django/utils/dateparse.py

```diff
@@ -29,7 +29,7 @@
 
 standard_duration_re = re.compile(
     r'^'
-    r'(?:(?P<days>-?\d+) )?'
+    r'(?:(?P<days>-?\d+) (days, )?)?'
```

Comment:
According to the [CPython source code](https://github.com/python/cpython/blob/master/Lib/datetime.py#L455), the "s" is there only if there's more than 1 day.